interfaces/builtin/tpm.go

```diff
@@ -0,0 +1,67 @@
+package builtin
+
+import "github.com/snapcore/snapd/interfaces"
+
+var tpmPermanentSlotAppArmor = []byte(`
+# Description: for those who need to talk to the system TPM chip over /dev/tpm0
+# Usage: common
+
+/dev/tpm0 rw,
+`)
+
+type TpmInterface struct { }
```

Comment:
Also we can simply this class a lot of the allow the common interface class to specify slot snippets too. @zyga what do you think?